applications/launchpad/gui-react/src/components/CoinsList/styles.ts

```diff
@@ -22,5 +22,6 @@ export const IconWrapper = styled.span`
     width: 24px;
     height: 24px;
   }
-  color: ${({ theme }) => theme.inverted.secondary};
+  /* color: ${({ theme }) => theme.inverted.secondary}; */
+  color: yellow;
```

Comment:
Is it temporary change? If so, we would need to add at least some `@TODO`, but ideally would be to revert this change.